spec/loyalty/_L_sign_in.test.js

```diff
@@ -1,29 +1,47 @@
-var loyalty = require("../../common/loyaltyFunctions")
-
 describe('Sparks Sign In Journey', function() {
 
     beforeEach(function() {
         browser.url('http://marksandspencer.com');
     })
 
+    function sparksPageVerification() {
```

Comment:
Move the functions to common files. In test files, we should keep only test functions. 
These functions are reusable and should be used as a module.